### PR TITLE
add wrapper for paladin protein mapper

### PR DESCRIPTION
**Pull request by bluegenes**
software: https://github.com/ToniWestbrook/paladin

paper: [PALADIN: protein alignment for functional profiling whole metagenome shotgun data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5423455/)

### CODE REVIEW

**Comment by bluegenes** [comment]
ready for review

File: bio/paladin/index/test/Snakefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+rule paladin_index:
+    input:
+        "prot.fasta", 
+    output:
+        "index/prot.fasta.bwt" 
+    log:
+        "logs/paladin/prot_index.log"
+    params:
+      reference_type=3
+    threads: 4
```
Comment (johanneskoester) [inline bio/paladin/index/test/Snakefile]: Seems like threads are not used in the wrapper.py?

File: bio/paladin/prepare/test/Snakefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+rule paladin_prepare:
+    output:
+        "uniprot_sprot.fasta.gz",
+        "uniprot_sprot.fasta.gz.pro" 
+    log:
+        "logs/paladin/prepare_sprot.log"
+    params:
+        reference_type=1, # 1=swiss-prot, 2=uniref90 
+    threads: 4
```
Comment (johanneskoester) [inline bio/paladin/prepare/test/Snakefile]: Again, threads seem to be unused in the wrapper.py.

File: bio/paladin/index/test/Snakefile (hunk 1 above)
Comment (bluegenes) [reply bio/paladin/index/test/Snakefile]: whoops,. seems like multithreading is not available. Thanks for catching!

**Pull request merged**